courses/free-courses-pl.md

```diff
@@ -86,6 +87,7 @@
 
 * [Kurs MySQL](https://www.youtube.com/playlist?list=PL748D0ACBEC371708)
 * [Kurs MySQL. Bazy danych, język zapytań SQL](https://www.youtube.com/playlist?list=PLOYHgt8dIdoymv-Wzvs8M-OsKFD31VTVZ) - Mirosław Zelent, Damian Stelmach
+* [Ms SQL Server – Podstawy](https://kursy.online-project.pl/product/ms-sql-server-podstawy) - Andrzej Śmigielski
```

Comment:
bad link?

Reply:
yes, sorry😢

Reply:
I've looked at this again. link works, but the whole ordering thing with price zero makes it hard to confirm its free. Have you taken the course?

Reply:
sorry for delays #9109